### DISCUSSION TITLE
Import / Export - Intune / Intune Manager JSON

### DISCUSSION THREAD

**Discussion by E-Coll** [Micke-K/IntuneManagement#196] [Ideas]
Hi

Is there already a possibility or is it being considered that you can use the new Intune Import / Export function for the Intune Manager Policies?

Unfortunately, it is currently not possible to import policies that have been exported by the Intune Manager via Intune.

Especially for the situation in which you only need 1x policy quickly, this would be very practical

**Comment by Micke-K** [comment]
Hello!

Nothing planned but this seams like a good idea. Should not be that hard except it might break some functionality. I have full OData information in the json. Could be that import in the portal does not like that. 

Cheers!
